### PR TITLE
Add PBJ package option

### PR DESCRIPTION
**Pull request by jasperpotts**
Add protobuf option to configure the new destination package for PBJ generated Java code.

### CODE REVIEW

File: mirror/consensus_service.proto [hunk 1]
```diff
@@ -21,6 +21,7 @@
 syntax = "proto3";
 
 package com.hedera.mirror.api.proto;
+option pbj_java_package = "com.hedera.hapi.mirror";
```
Comment (steven-sheehy) [inline mirror/consensus_service.proto:24]: Why are we putting mirror node API protos under a HAPI folder? I thought we said pbj would exclude mirror protos?

Comment (rbair23) [reply mirror/consensus_service.proto:24]: Same question for the SDK.

Comment (jasperpotts) [reply mirror/consensus_service.proto]: Good question, can probably remove from SDK and Mirror. I was mostly adding for completeness. They were going to be excluded in hedera-protobufs-java build. What is the preference, leave or remove?

File: README.md [hunk 2]
```diff
@@ -36,9 +40,22 @@ must be paid in ℏ from a cryptocurrency account. The payer authorizes a
...
+## Branching
 This repository uses a simple branching model with only two distinguished branches:
  1. The head of `main` points to the latest protobufs blessed for deployment to mainnet. 
 Tags such as `v0.12.0` mark commits used for testnet and mainnet deploys.
  2. The head of `develop` points to the latest candidate for the next tag in `main`. 
 Tags suffixed with `-alpha.x` mark commits used for previewnet deploys. 
+
+## PBJ Package Support
+Proto files in this repo are generated into two kinds of Java files, one using **protoc** and other using 
+[PBJ](https://github.com/hashgraph/pbj) a custom java code generation library for hedera. To support PBJ all proto files
```
Comment (steven-sheehy) [inline README.md:54]:
```suggestion
[PBJ](https://github.com/hashgraph/pbj) a custom java code generation library for Hedera. To support PBJ all proto files
```

**Review by rbair23 — APPROVED** [review]

**Review by steven-sheehy — APPROVED** [review]

**Review by kimbor — APPROVED** [review]

**Pull request merged**